.github/workflows/gauntlet.yml

```diff
@@ -61,5 +61,18 @@ jobs:
         with:
           name: chainlink-cosmos
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+      - name: Cache Wasmd Artifacts
+        uses: actions/cache@v3
+        id: cache-wasmd-artifacts
+        env:
+          cache-name: cache-wasmd-artifacts
+        with:
+          path: artifacts
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('contracts') }}
```

Comment:
Here's what my root artifacts/ folder looks like after running local tests ~

<img width="295" alt="image" src="https://github.com/smartcontractkit/chainlink-cosmos/assets/14297860/7e3be13d-e216-4d93-994b-ab52a1383835">

I'm pretty confident that the cw-plus contracts are never stored in the artifacts/ folder in the default workflow because it is read as an array buffer.

The way it's written, I think the original author intended for cw-plus contracts to be downloaded (https://github.com/smartcontractkit/chainlink-cosmos/blob/861c937bfb8536e463fc3dc4dd43b57e0e4138b7/packages-ts/gauntlet-cosmos-contracts/src/lib/contracts.ts#L130) whereas chainlink contracts are found locally because cw-plus contracts never have the version set to 'local'